concent_api/core/tests/test_integration_subtask_results_verify.py

```diff
@@ -297,15 +298,14 @@ def test_that_concent_accepts_valid_request_and_sends_verification_order_to_work
 
         # when
         with mock.patch("core.message_handlers.base.is_account_status_positive", return_value=True) as is_account_status_positive_mock:
-            with mock.patch("core.message_handlers.send_blender_verification_request") as send_verification_request_mock:
-                with freeze_time(subtask_results_verify_time_str):
```

Comment:
why this mock has been removed?

Reply:
Okey, I've change it to more specific `mock.patch("core.queue_operations.blender_verification_request.delay")`